proto/viam/app/datapipelines/v1/data_pipelines.proto

```diff
@@ -68,7 +84,22 @@ message ListDataPipelinesResponse {
 }
 
 message CreateDataPipelineRequest {
-  DataPipeline data_pipeline = 1;
+  // The associated Viam organization ID.
+  string organization_id = 1;
+
+  // A unique identifier at the org level.
+  string name = 2;
+
+  // A MongoDB aggregation pipeline as a list of BSON documents, where
+  // each document is one stage in the pipeline.
+  repeated bytes mql_binary = 3;
+
+  // A cron expression representing the expected execution schedule in UTC (note this also
+  // defines the input time window; an hourly schedule would process 1 hour of data at a time).
+  string schedule = 4;
+
+  // Whether or not the pipeline is enabled.
+  bool enabled = 5;
```

Comment:
Do we want pipelines to be enabled on creation? Or do we want to put the onus on the user to specify?